MVP_Clicker/Assets/Project/Scripts/Game/Areas/Skill/Presenter/SkillPresenter.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using Project.Scripts.Game.Areas.Skill.Config;
+using Project.Scripts.Game.Areas.Skill.Model;
+using Project.Scripts.Game.Areas.Skill.View;
+
+namespace Project.Scripts.Game.Areas.Skill.Presenter
+{
+    public class SkillPresenter : IDisposable
+    {
+        private readonly ISkillModel _model;
+        private readonly ISkillView _view;
+        private readonly ISkillConfig _config;
+
+        public SkillPresenter(ISkillModel model, ISkillView view, ISkillConfig config)
+        {
+            _model = model;
+            _view = view;
+            _config = config;
+
+            _view.Description = config.Description;
+            _view.Name = config.Name;
+            _view.RecoveryDuration = config.RecoveryDuration;
+            _view.SkillIcon = config.SkillIcon;
+            AddListeners();
+        }
+
+        private void Activate()
+        {
+            _model.TryActivate();
+        }
+
+        private void AddListeners()
+        {
+            _view.Activated += Activate;
+            _model.UpdatedCooldown += _view.SetShadowIntensityOnSkill;
+        }
+
+        private void RemoveListeners()
+        {
+            _view.Activated -= Activate;
+            _model.UpdatedCooldown -= _view.SetShadowIntensityOnSkill;
```

Comment:
When you subscribe to events it's more good to create a new method and there invoke the specific logic that you need.
_model.UpdatedCooldown -= OnUpdatedCooldown;

There are few reasons.
One of them, that it's make clear to brain that this is behaviour of subscribtion to event.
I mean that brain understand that this is not the such part of code:
```
int some1 = 5;
int some2 = 6;
some1 += some2;
```
when we have some method with prefix 'On' it's more clear/

Second reason, that sometimes events has parameter and sometimes we dont want to use this parameter. Exemple
```
public event Action<int> MyEvent;
// ....
MyEvent += OnMyEvent;
// ....
public void OmMyEvent(int myInt)
{
     CallMyMethodWithoutParameter();
}
```